src/routes/_components/dialog/components/MediaDialog.html

```diff
@@ -263,12 +302,14 @@
     },
     store: () => store,
     data: () => ({
-      pinchZoomMode: false
+      pinchZoomMode: false,
+      showFullImageDescription: false
     }),
     computed: {
       length: ({ mediaItems }) => mediaItems.length,
       dots: ({ length }) => times(length, i => ({ i })),
       canPinchZoom: ({ mediaItems }) => !mediaItems.some(media => ['video', 'audio'].includes(media.type)),
+      hasDescription: ({ mediaItem }) => mediaItem.description,
```

Comment:
A bit more clear that this is a boolean:

```suggestion
      hasDescription: ({ mediaItem }) => !!mediaItem.description,
```

Reply:
Good call - I slightly prefer `Boolean()` in these situations, but I think either is fine if you have a style preference.

Reply:
`Boolean()` is fine too, yeah.